documentation/user_docs/docs/user_guide/Advanced_use.rst

```diff
@@ -108,9 +138,29 @@ which gives us a structure array called ``array_out`` which has the following fi
        npix: [41x101 double]
 
 
-It should be fairly clear what most of the fields of this structure array are, but it is useful to point out a couple of important ones now. ``ulen`` gives the conversion factor between reciprocal lattice units and :math:`\\AA^{-1}`. ``pax`` and ``iax`` tell us that the parameter axes are the 1st and 4th, and that the integration axes (i.e the fixed ones) are the 2nd and 3rd. The fields ``s`` and ``e`` are arrays which contain the scattering signal and the variance (i.e. the square of the errobar that is plotted). These arrays are filled with zeros when the object is created in this manner. The cell array called ``p`` contains two vectors which specify the bin boundaries of the pixels which were specified during the object creation. The ``pax`` and ``dax`` vectors respectively specify which of the axes (from the columns of ``u_to_rlu``) are variables (as opposed to being integrated over), and which way round the axes will be when plotted. Finally there is the array called npix. This tells us whether a pixel is contributing to the scattering, so it is 1 when the corresponding element of ``s`` is a number, and is zero if the corresponding element of ``s`` is NaN.
+It should be fairly clear what most of the fields of this structure array are, but it is useful to point out a couple of
+important ones now:
+
+- ``ulen`` gives the conversion factor between reciprocal lattice units and :math:`\\AA^{-1}`.
+
+- ``pax`` and ``iax`` tell us that the parameter axes are the 1st and 4th, and that the integration axes (i.e the fixed
```

Comment:
I think at least these two are out of date after Alex' changes but please ask him